core/src/apps/kaspa/sign_tx.py

```
@@ -62,25 +63,31 @@ async def sign_tx(
     return KaspaSignedTx(signature=signature)
 
 
-async def sign_input(ctx, msg: Signable, keychain) -> bytes:
+async def sign_input(ctx, msg: Signable, keychain, use_tweak: bool = True) -> bytes:
```

Ruff (0.11.9)

Missing type annotation for function argument `ctx`

(ANN001)